subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r36/BuildProgressCrossVersionSpec.groovy

```diff
@@ -73,4 +83,311 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
         events.operation('Task :runInProcess').descendant('My in-process worker action')
         events.operation('Task :runForked').descendant('My forked worker action')
     }
+
+    @ToolingApiVersion(">=3.6")
```

Comment:
Should be `@TargetGradleVersion`, there's no changes to the tooling API. The test should describe which versions of Gradle and the tooling API that the feature will work with, not which versions it should be tested against - that's decided by something else.

Same for the other methods of this test class.